workday/15.0/modules/ROOT/pages/workday-reference.adoc

```diff
@@ -0,0 +1,2954 @@
+** HOURS
+** DAYS |  Time unit for Response Timeout field |  |
+| Target Variable a| String |  The name of a variable that stores the operation's output |  |
+| Target Value a| String |   An expression that evaluates against the operation's output. The outcome of the expression is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy for connectivity errors |  |
+|======================
+
+==== Output
+[cols=".^50%,.^50%"]
+|======================
+| *Type* a| Binary
+| *Attributes Type* a| <<SoapAttributes>>
+|======================
+
+==== For Configurations.
+* <<config>>
+
+==== Throws
+* WORKDAY:CONNECTIVITY
+* WORKDAY:PROCESSING_ERROR
+* WORKDAY:RETRY_EXHAUSTED
+* WORKDAY:VALIDATION_ERROR
+
+
+[[notification]]
+=== Notification
+`<workday:notification>`
+
+
+Invoke the Notification web service, which provides an operation that informs external systems about business events that occur within Workday. The WSDL for this service provides the structure that an external system must implement to receive notifications for their subscribed Workday business events. Subscriptions and notification details, such as endpoint and security information, are defined within the Workday application.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Operation a| String |  Type of operation to execute. |  | x
+| Content a| Binary |  Payload for the operation. |  `#[payload]` |
+| Headers a| Binary |  Headers included in the request. |  |
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
+| Response Timeout a| Number |  How long to wait for a response, based on the unit in the Response Timeout field  |  |
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the Response Timeout field |  |
+| Target Variable a| String |  The name of a variable that stores the operation's output. |  |
+| Target Value a| String |  An expression that evaluates against the operation's output. The outcome of the expression is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy for connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+|Attributes Type | <<SoapAttributes>>
+|===
+
+=== For Configurations
+* <<config>>
+
+==== Throws
+* WORKDAY:CONNECTIVITY
+* WORKDAY:PROCESSING_ERROR
+* WORKDAY:RETRY_EXHAUSTED
+* WORKDAY:VALIDATION_ERROR
+
+
+[[OrgStudio]]
+== Org Studio
+`<workday:org-studio>`
+
+
+Web service operations designed for the interaction with Org Studio Setup Data.
```

Comment:
Suggestion: Web service operations designed for interacting with Org Studio setup data.